sdk/spring-sdk/src/test/java/kalix/springsdk/valueentity/RestAnnotatedValueEntities.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.valueentity;
+
+import kalix.javasdk.valueentity.ValueEntity;
+import kalix.springsdk.annotations.EntityKey;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+public class RestAnnotatedValueEntities {
+
+  @EntityKey("id")
```

Comment:
Could it make sense a convention with a `entityKey` path parameter being the id so that this annotation is optional and only needed if some other name is used, less noise (but a bit more magic)?

Reply:
The problem is that if people get it wrong, they won't know it until deploying and trying to hit the entity.

Maybe we could enforce it as well. Like any entity type must have a paths containing `entityKey` or have the annotation.

Reply:
Let's explore this on another PR. We can probably enforce it and eventually catch it at compile time with annotation processors.

Btw, at least for entities, we will need some annotation at type level. We need to have the `entityType` string. Can be `@ValueEntity("foo")` or generic one `@KalixComponent("foo")`.